frontend/src/pages/components/icons.js

```diff
@@ -0,0 +1,551 @@
+// +-------+------------------------------------------------------------------------
+// | Notes |
+// +-------+
+
+/******************************************************************************
+ Icons defines icons used in the app:
+    AddImageToAlbumIcon,
+    AddImagesIcon,
+    AnimationIcon,
+    CodeIcon,
+    CommentIcon,
+    DeleteAlbumIcon,
+    MoreIcon,
+    PrivacyIcon,
+    SaveIcon,
+    ShareIcon,
+    StarIcon
+*********************************************************************************/  
+
+// +---------+----------------------------------------------------------------------
+// | Imports |
+// +---------+
+
+import React, { useContext, useState, useRef} from "react";
+import {
+  Button, Row,
+  Col, Nav, NavDropdown, Popover, Overlay, Form,
+  Modal, OverlayTrigger
+} from "react-bootstrap";
+import { UserContext } from './Contexts/UserContext';
+import {Link, useLocation } from "react-router-dom";
+import "../../design/styleSheets/gallery.css";
+import TimeAgo from 'javascript-time-ago'
+import en from 'javascript-time-ago/locale/en'
+import copy from 'copy-to-clipboard';
```

Comment:
This should be added as a dependency to the package.json

Reply:
Thanks! We'll make sure to add copy-to-clipboard.